toolchain/check/import_ref.cpp

```diff
@@ -676,6 +676,69 @@ static auto AddImportIRInst(ImportContext& context, SemIR::InstId inst_id)
       {.ir_id = context.import_ir_id(), .inst_id = inst_id});
 }
 
+// Sets the constant value for an instruction and returns it. The constant value
+// is computed if not explicitly specified.
```

Comment:
Computation is no longer conditional:

```suggestion
// Computes, sets, and returns the constant value for an instruction.
```